### PR TITLE
Automate fixing of install yaml

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

**Comment by onyiny-ang** [comment]
Test failing here too #159. Is this being tracked/addresses elsewhere?

**Comment by marun** [comment]
Rebased on #162

**Comment by onyiny-ang** [comment]
@marun @font distinguishing the release artifact name from the kubebuilder generated artifact name makes sense to me too.
/lgtm for the rest of the PR too.

File: scripts/download-binaries.sh [hunk 1]
```diff
@@ -56,21 +52,17 @@ curl "${curl_args}O" "${kb_url}" \
...
-# compatible.
+# Use a stable version of kube-apiserver to ensure a version >= 1.11.
 # TODO(marun) Remove when kubebuilder includes released 1.11 binaries
-ci_version="$(curl "$curl_args" https://storage.googleapis.com/kubernetes-release-dev/ci/latest-green.txt)"
-ks_tgz="kubernetes-server-linux-amd64.tar.gz"
-ks_url="https://storage.googleapis.com/kubernetes-release-dev/ci/${ci_version}/${ks_tgz}"
-curl "${curl_args}O" "${ks_url}" \
-  && tar xzf "${ks_tgz}" kubernetes/server/bin/kube-apiserver -C "${dest_dir}" --strip-components=2 \
-  && rm "${ks_tgz}"
+stable_version="$(curl "${curl_args}" https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
```
Comment (gyliu513) [inline scripts/download-binaries.sh]: This should already be merged in https://github.com/kubernetes-sigs/federation-v2/pull/162/files

Comment (onyiny-ang) [reply scripts/download-binaries.sh]: @gyliu513 you'll notice that there is reference to this being merged in the commit header. The first two commits of this PR are from #162

**Comment by shashidharatd** [comment]
this is mergable. lgtm. Thanks @marun !

**Pull request merged**